src/api/firebase.js

```diff
@@ -63,14 +69,37 @@ export async function addItem(listId, { itemName, daysUntilNextPurchase }) {
 	});
 }
 
-export async function updateItem() {
+export async function updateItem(listId, itemData) {
+	const docRef = doc(db, listId, itemData.id);
+
+	await updateDoc(docRef, {
+		isChecked: !itemData.isChecked,
+		dateLastPurchased: new Date(Date.now()),
+		totalPurchases: itemData.totalPurchases,
+	});
+
 	/**
+
 	 * TODO: Fill this out so that it uses the correct Firestore function
 	 * to update an existing item! You'll need to figure out what arguments
 	 * this function must accept!
-	 */
+**/
 }
 
+// export async function updateItem(db, listId, itemData, ) {
+// 	const docRef = doc(db, itemData, listId);
+// 	await updateDoc(docRef, itemData)
+// 	{/**
+// 	   const newCityRef = doc(collection(db, "cities"));
+// 	 * @param database → db
+// 	 * @param collection name → cities
+// 	 * @param document ID
+// 	 * TODO: Fill this out so that it uses the correct Firestore function
```

Comment:
I think this note could be deleted?

Reply:
line 92- 96 was removed before the pull. Thank you